spec/system/candidate_interface/references/reference_selection/candidate_selects_two_references_spec.rb

```diff
@@ -6,27 +6,35 @@
   scenario 'the candidate has received 4 references and must select 2 before completing the section' do
     given_i_am_signed_in
     and_the_reference_selection_feature_flag_is_active
-    and_i_have_requested_four_references
```

Comment:
Worth covering what happens when no references are received yet and the candidate navigates to the select references page.